src/frontend/components/SøknadsSteg/Forside/FortsettPåSøknad.tsx

```diff
@@ -33,6 +30,9 @@ const ModalInnholdContainer = styled.div`
     flex-direction: column;
     padding: 1rem 2rem;
     max-width: 35rem;
+    > h1 {
```

Comment:
Hva er dette? Er det for å endre TekstBlock. I så fall kunne vi gjort det på TekstBlock så slipper vi å fikse det alle steder vi bruker TekstBlock